### ISSUE TITLE
Force close channel not working as intended. 

### ISSUE THREAD

**Issue by destilot** [LN-Zap/zap-desktop#3766]
I Opened a channel with Zap Technologies on Jun 13, 2021, 2:26 AM (190,950 Satoshis)
Funding Transaction: https://blockstream.info/tx/97c6dd726b9bd346b2fed442575544b028563ecb811360446bb732783e25bb24
Approximately one week ago, I discovered that the channel was showing Zap Technologies as Offline. I then used the Force close channel function on Sep 2, 2021, 11:00 AM. I selected Fee: Fast. Mempool has been empty more than once in this time so fee should not be the problem. At the same time a close another channel the normal way and that didn’t give me any problems.  
Now the channel to ZAP show status: pending close. If I use the “filter” in the channels view, it is  listed under offline. 
I get that there is a timeclock of 144 block approximately 24 hours. 
I am now LN expert, and any help will be appreciated. Is there anyway to recover the Sats? 

Info about my client:  
Zap 0.7.4-beta
Windows: 
Edition: Windows 10 Enterprise
Version: 21H1
OS build: 19043.1165
Experience: Windows Feature Experience Pack 120.2212.3530.0

Screenshots:
![1](https://user-images.githubusercontent.com/72632886/132142138-3c2ecae1-3e98-41ca-ab03-fad2a62ff509.JPG)
![2](https://user-images.githubusercontent.com/72632886/132142139-9ee9416b-123b-4bc1-bafb-f6e3489c325e.JPG)
![3](https://user-images.githubusercontent.com/72632886/132142140-b6bfc29c-60bb-4b96-9c96-f612a1a83115.JPG)
![4](https://user-images.githubusercontent.com/72632886/132142141-f93e0992-6a76-48c1-bc3f-b4f4d87c5185.JPG)
![5](https://user-images.githubusercontent.com/72632886/132142142-3c3095fb-d8d7-427d-8cc9-c6c02e987b0a.JPG)
![6](https://user-images.githubusercontent.com/72632886/132142143-98e36336-b155-4789-a33c-dfd2cca3e8d0.JPG)
![7](https://user-images.githubusercontent.com/72632886/132142144-e3e95022-ac87-4d10-98b2-9fff15f7830f.JPG)

**Comment by Lonetracke** [comment]
Also.
Offline>Forse-Close>TIME>Logout>Offline......

**Comment by destilot** [comment]
Yes. Same here, if I close the wallet and re-open it the channel is back to offline

**Comment by umlautmatt** [comment]
I'm having the exact same problem

**Comment by destilot** [comment]
Think this problem effect all windows ZAP users. Hope they will fix it in the next release. I have not gotten any response from the team, only from users with the same problem.

**Comment by umlautmatt** [comment]
Hmm I'm using Mac OS

**Comment by destilot** [comment]
Interesting, then it is an even bigger problem.

**Comment by satyakamc** [comment]
> I Opened a channel with Zap Technologies on Jun 13, 2021, 2:26 AM (190,950 Satoshis) Funding Transaction: https://blockstream.info/tx/97c6dd726b9bd346b2fed442575544b028563ecb811360446bb732783e25bb24 Approximately one week ago, I discovered that the channel was showing Zap Technologies as Offline. I then used the Force close channel function on Sep 2, 2021, 11:00 AM. I selected Fee: Fast. Mempool has been empty more than once in this time so fee should not be the problem. At the same time a close another channel the normal way and that didn’t give me any problems. Now the channel to ZAP show status: pending close. If I use the “filter” in the channels view, it is listed under offline. I get that there is a timeclock of 144 block approximately 24 hours. I am no LN expert, and any help will be appreciated. Is there anyway to recover the Sats?
> 
> Info about my client: Zap 0.7.4-beta Windows: Edition: Windows 10 Enterprise Version: 21H1 OS build: 19043.1165 Experience: Windows Feature Experience Pack 120.2212.3530.0
> 
> Screenshots: ![1](https://user-images.githubusercontent.com/72632886/132142138-3c2ecae1-3e98-41ca-ab03-fad2a62ff509.JPG) ![2](https://user-images.githubusercontent.com/72632886/132142139-9ee9416b-123b-4bc1-bafb-f6e3489c325e.JPG) ![3](https://user-images.githubusercontent.com/72632886/132142140-b6bfc29c-60bb-4b96-9c96-f612a1a83115.JPG) ![4](https://user-images.githubusercontent.com/72632886/132142141-f93e0992-6a76-48c1-bc3f-b4f4d87c5185.JPG) ![5](https://user-images.githubusercontent.com/72632886/132142142-3c3095fb-d8d7-427d-8cc9-c6c02e987b0a.JPG) ![6](https://user-images.githubusercontent.com/72632886/132142143-98e36336-b155-4789-a33c-dfd2cca3e8d0.JPG) ![7](https://user-images.githubusercontent.com/72632886/132142144-e3e95022-ac87-4d10-98b2-9fff15f7830f.JPG)

Could you get back your BTC after the time lock period of 144 blocks?

**Comment by destilot** [comment]
No, I did not. The problem is in the lates version of Zap as well

**Comment by satyakamc** [comment]
That's surprising, your issue has been there since Sep 2021...No one from ZAP is responding to it. And there are many more like you...

**Comment by mrfelton** [comment]
It's possible that your closing transaction was not broadcast (bugs inside lnd sometimes cause this to happen) or that it was broadcast with a fee that was too low, causing it to not confirm and to eventually fall out of the mempool.

Can you provide the closing channel tx id?

**Comment by destilot** [comment]
Tnx for the response. @mrfelton - The fee was 9005 Sats and the mempool was empty at the time. But the broadcast bug can be the reason. How can I check the channel tx id in the Zap wallet or logs?

**Comment by PHulsebosch** [comment]
I have the same issue. Also my funds are stuck in the LN. Is this a scam, or a bug. This shows to me that the LN is not yet ready for use with real bitcoins.

**Comment by PHulsebosch** [comment]
Just looking again in my ZAP and the channel is closed and my funds are back. I don't think it is because of my commend 10 minutes ago, but just as it should be in the LN code. Getting more confidence in that again. Keep up with the good work!

**Comment by harrymmmm** [comment]
I've had 90000 sats stuck in a channel with zap as described above for about 2 years now. The channel is (now at least) permanently offline and force close attempts are unsuccessful. 
Not panicking about $20 or so, but if it's easy here's the funding transaction: b8375a9f68283ea512a5af2188c5aaad0c4f3058a6d70202817efae6f3f2573f

I had a second channel opened for a long time but in that case was lucky enough to catch the channel online. A cooperative close worked in that case.

**Comment by oddaran** [comment]
Hey @harrymmmm Id like to know how you were able to resolve the channel issue that is/was unsuccessful. I've got a similar problem but it's for a higher amount of btc. Please contact me at oddaran@gmail.com. Thanks.

**Comment by harrymmmm** [comment]
@oddaran I had pretty much given up on it when I posted that message. When I got no replies I just gave up.

To see if the channel status had changed now, I just tried running the appimage I was running at the time and got this error - I guess I'd need to get a later version:
$ ./Desktop/appimages/Zap-linux-x86_64-v0.7.6-beta.AppImage
...
[195449:0316/043308.958140:FATAL:gpu_data_manager_impl_private.cc(439)] GPU process isn't usable. Goodbye.
Trace/breakpoint trap (core dumped)
